### ISSUE TITLE
"Enable DXVK" toggle is misbehaving when set to "Manual" in 0.4.21

### ISSUE THREAD

**Issue by Nocifer** [lutris/lutris#1237]
I'm on Arch, and I have DXVK installed manually. In Lutris, I keep "Enable DXVK" toggled on and the version set to "Manual", and until today all was well.

With the newest Lutris 0.4.21 release, trying to launch a game in this way results in the following errors, one after the other:

![2018-10-23 16-53-41](https://user-images.githubusercontent.com/6195422/47365526-71818000-d6e4-11e8-9235-43bfac59e795.png)
![2018-10-23 16-53-51](https://user-images.githubusercontent.com/6195422/47365542-79d9bb00-d6e4-11e8-924e-03ba21f1a90f.png)

Toggling "Enable DXVK" off results in Lutris deleting everything DXVK-related in Wine's system32/syswow64 folders, and of course produces the following error:

![2018-10-23 17-00-55 error](https://user-images.githubusercontent.com/6195422/47365963-59f6c700-d6e5-11e8-93ef-488285e83256.png)

Toggling "Enable DXVK" on and changing the version to something else works as expected, but then I'm forced to use the DXVK release catalog that Lutris provides, which can lag significantly behind official DXVK releases. But even if this matter were resolved in some manner (e.g. by introducing a "Latest" DXVK version as per #1207), and even if the solution suited my tastes, I would still like to have the option to use my own binaries if I so choose.

In other words, please fix the "Manual" version :)

**Comment by sigmaSd** [comment]
https://github.com/lutris/lutris/blob/master/lutris/gui/config_boxes.py#L254 prioritize with `has_entry` flag set to true will prioritize value text over choice and hence the issue

**Comment by sigmaSd** [comment]
Maybe this should just be treated symptomatically , for example change manual to Manual,  I think changing config_boxes logic to handle this is too troublesome

**Comment by Nocifer** [comment]
Alright, that makes sense. What doesn't make sense, though, is that "manual" seems to actually work! Why would that be, when everything else triggers that runtime error..?

**Comment by sigmaSd** [comment]
It's not complicated, you are given the option "Manual" that correspond in the code to value "manual", the bug is that the gui input box consider "Manual" not a choice that it needs to lookup its value but the value  itself, if u're curious just check lutris/runners/wine.py I feel like I'm not explaining it well

**Comment by Nocifer** [comment]
Ah, now I see. So yeah, a quick "symptomatic" fix of replacing "Manual" with "manual" would probably be a good solution, since we're talking only about this one specific case. But given that it's only one specific case, could a better solution down the road be, rather than having "Manual" or "manual" or whatever else as a choice in the version drop-down, to instead have a separate toggle for it? Something like this:

![2018-10-24 13-09-22 configure wine](https://user-images.githubusercontent.com/6195422/47426318-8ec25700-d795-11e8-84e2-6d185379dc9e.png)

Just a thought for the middle- or long-term future :)

**Comment by sigmaSd** [comment]
nice idea and cool mockup :+1:

**Comment by sigmaSd** [comment]
Just so you know it is indeed a mid long term feature (aka not easy) , because right now its hard for those gui elements  to interact with each other with current lutris code base

**Comment by strycore** [comment]
This is overcomplicating the UI for no reason, I'll just fix the "manual" code.
